plugins/publish-changes/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "strapi-plugin-publish-changes",
+  "version": "0.0.0",
+  "description": "This is the description of the plugin.",
+  "strapi": {
+    "name": "publish-changes",
+    "icon": "plug",
+    "description": "Launch a GitHub Action from Strapi."
+  },
```

Comment:
Maybe we could put something useful here and choose a cool icon

Reply:
Changed icon and description